src/java.base/share/classes/com/sun/crypto/provider/HmacCore.java

```diff
@@ -22,6 +22,11 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2023, 2023 All Rights Reserved
+ * ===========================================================================
+ */
```

Comment:
This is only appropriate if some of the content is contributed by IBM: please remove these lines.

Reply:
Removed